bionic/flow.py

```diff
@@ -1209,6 +1249,7 @@ def __init__(self, state, _official=False):
 
         self.get = ShortcutProxy(self.get)
         self.setting = ShortcutProxy(self.setting)
+        self.entity_docstring = ShortcutProxy(self.entity_docstring)
```

Comment:
Actually I was thinking we could put this docstring on the `ShortcutProxy` itself, so that when you type `flow.get.my_entity?` or `flow.get.my_entity(<Tab>` you can see it in your notebook/IDE.

In general, I'm not sure many people will read the docstrings by explicitly calling `flow.entity_docstring()`.  However, we can attach them to places where user already knows to look for information, like on these methods, in the DAG visualization, perhaps in auto-generated Sphinx docs for a flow, etc.